frontend/.snyk

```diff
@@ -16,10 +16,14 @@ ignore:
         expires: '2019-06-12T23:18:30.879Z'
   SNYK-JS-LODASH-450202:
     - newrelic > async > lodash:
-        reason: 'Upgrade path did not work, patching for now'
-        expires: '2020-01-25T16:18:13.016Z'
+        reason: ignore for 30 days
+        expires: '2019-11-01T18:45:56.815Z'
   SNYK-JS-MARKED-451341:
     - ngx-md > marked:
         reason: no upgrade available
         expires: '2020-01-25T23:02:43.714Z'
+  SNYK-JS-HTTPSPROXYAGENT-469131:
+    - newrelic > https-proxy-agent:
+        reason: ignore for 30days
+        expires: '2019-11-01T18:45:56.816Z'
```

Comment:
Please push this expiration out to `2020-01-25` to match the others so that we don't need to fix this again tomorrow.

Reply:
done. Thanks for catching up!!